server/src/main/java/org/apache/uniffle/server/RegisterHeartBeat.java

```diff
@@ -58,11 +54,12 @@ public RegisterHeartBeat(ShuffleServer shuffleServer) {
     CoordinatorClientFactory factory = CoordinatorClientFactory.getInstance();
     this.coordinatorClients =
         factory.createCoordinatorClient(
-            conf.get(ShuffleServerConf.RSS_COORDINATOR_CLIENT_TYPE), this.coordinatorQuorum);
+            conf.get(ShuffleServerConf.RSS_COORDINATOR_CLIENT_TYPE),
+            this.coordinatorQuorum,
+            0,
```

Comment:
The `0` could be as the default value in the initial `createCoordinatorClient`

Reply:
public synchronized CoordinatorGrpcRetryableClient createCoordinatorClient(
      ClientType clientType,
      String coordinators,
      int heartBeatThreadNum)    
would conflict with
public synchronized CoordinatorClient createCoordinatorClient(
      ClientType clientType, String host, int port) ,
so it's not convenient to add

Reply:
Ok